posts/2020-11-09-ocp-log-forwarding.adoc

```diff
@@ -2,33 +2,33 @@
-blog_description: Using Log Forwarding API, you can send logs from Open Liberty deployed in OpenShift Container Platform to remote destinations. You can deploy Splunk on your external machine or inside your OpenShift Container Platform. The logs can be forwarded to Splunk using the Fluentd forward protocol.
-additional_authors: 
+author_picture: https://avatars3.githubusercontent.com/melissa-lee
+author_github: https://github.com/melissa-lee
+seo-title: Forwarding Open Liberty logs in OpenShift to Splunk by using the Log Forwarding API - OpenLiberty.io
+seo-description: Using the Log Forwarding API, you can send logs from Open Liberty that are deployed in OpenShift Container Platform to remote destinations. You can deploy Splunk on your external machine or inside your OpenShift Container Platform. The logs can be forwarded to Splunk using the Fluentd forward protocol.
+blog_description: Using the Log Forwarding API, you can send logs from Open Liberty deployed in OpenShift Container Platform to remote destinations. You can deploy Splunk on your external machine or inside your OpenShift Container Platform. The logs can be forwarded to Splunk using the Fluentd forward protocol.
+additional_authors:
 - name: David Chan
   github: https://github.com/Channyboy
   image: https://avatars0.githubusercontent.com/Channyboy
 ---
-= Forwarding Open Liberty logs in OpenShift Container Platform to Splunk using Log Forwarding API
+= Forwarding Open Liberty logs in OpenShift to Splunk by using the Log Forwarding API
 Halim Lee <https://github.com/leemelim>
 
-OpenShift Container Platform provides a log aggregation solution with cluster logging using Elasticsearch, Fluentd, and Kibana (EFK stack). In addition to EFK stack, you can integrate with other log analysis solutions. Starting in OpenShift 4.6 you can use Log Forwarding API to send logs from Open Liberty in OpenShift cluster to other available log analysis solutions.  The Log Forwarding API is also available for use in OpenShift 4.3 to 4.5 under Technology Preview.
+Collecting logs in one location helps you monitor all of your applications efficiently, which is especially important in highly distributed and dynamic container orchestration systems. Red Hat OpenShift Container Platform (OCP) provides a log aggregation solution that uses Elasticsearch, Fluentd, and Kibana, also known as the EFK stack. In addition to the EFK stack, you can use the https://docs.openshift.com/container-platform/4.4/logging/config/cluster-logging-external.html?extIdCarryOver=true&sc_cid=701f2000001Css5AAC#cluster-logging-collector-log-forward_cluster-logging-external[Log Forwarding API] to send logs from Open Liberty in an OpenShift cluster to other log analysis solutions. The Log Forwarding API is also available starting with OpenShift 4.6, and in OpenShift 4.3 to 4.5 as a Technology Preview feature.
```

Comment:
The link used should point to the latest version (4.6). Which is the following link instead (they renamed their pages for 4.6)

https://docs.openshift.com/container-platform/4.6/logging/cluster-logging-external.html#cluster-logging-collector-log-forwarding-about_cluster-logging-external

Reply:
fixed